packages/web-app-files/src/mixins/spaces/actions/setReadme.js

```diff
@@ -0,0 +1,63 @@
+import { isLocationSpacesActive } from '../../../router'
+import { mapMutations, mapState, mapActions } from 'vuex'
+import { bus } from 'web-pkg/src/instance'
+
+export default {
+  computed: {
+    ...mapState('Files', ['currentFolder']),
+    $_setSpaceReadme_items() {
+      return [
+        {
+          name: 'set-space-readme',
+          icon: 'markdown',
+          handler: this.$_setSpaceReadme_setReadmeSpace,
```

Comment:
Same here, please stay consistent with the `trigger` naming:
```suggestion
          handler: this.$_setSpaceReadme_trigger,
```